src/helper/colors.ts

```diff
@@ -1,6 +1,7 @@
 export const colors = {
   brand: '#007ec1',
   brandGreen: '#95bc1a',
+  brandGreen300: '#B9D957',
```

Comment:
What does `brandGreen300` mean? Is there a better name?!

Reply:
It's a tailwind convention I adopted more of in this PR.

Reply:
See https://tailwindcss.com/docs/customizing-colors